instance/static/html/instance/index.html

```diff
@@ -0,0 +1,73 @@
+<top-bar>
+  <ul class="title-area">
+    <li class="name">
+      <h1><a href="#">OpenCraft - Instances Manager</a></h1>
+    </li>
+  </ul>
+
+  <section class="top-bar-section">
+    <!-- Right Nav Section -->
+    <ul class="right">
+      <li class="active"><a href="/admin/logout/?next=/">Logout</a></li>
+    </ul>
+  </section>
+</top-bar>
+
+<div class="row">
+  <!-- Instances List -->
+  <div class="large-3 columns">
+    <ul class="side-nav" ng-if="instanceList">
+      <li ng-repeat="instance in instanceList"
+          ng-class="{active: instance == selected.instance}">
+        <a ng-click="select('instance', instance)">{{ instance.name }}</a>
+      </li>
+      <li class="divider"></li>
+    </ul>
+
+    <div class="row" ng-if="!instanceList">
+      <p>...</p>
```

Comment:
@antoviaque this is totally confusing - first time I got to the instance manager I thought it was broken, because it was loading instances in the background without any visual hint. I believe having something like "Loading instances, please wait..." here would be helpful.